src/frontend/components/DashboardConfirmButton/index.tsx

```diff
@@ -27,7 +27,7 @@ export const DashboardConfirmButton = ({
   return (
     <React.Fragment>
       <DashboardButton
-        primary={true}
+        primary={false}
```

Comment:
Why ?

Reply:
I needed a failing test to try the artifacts in circle-ci.
It has been removed now.